Libplanet.Extensions.Cocona/Commands/StoreCommand.cs

```diff
@@ -26,7 +27,58 @@ public class StoreCommand
                     ["monorocksdb"] = storePath => new MonoRocksDBStore(storePath),
                 }.ToImmutableSortedDictionary();
 
-        [Command(Description = "Query a block by index")]
+        [Command(Description = "Build an index for transaction id and block hash.")]
+        public void BuildIndexTxBlock(
+            [Argument("STORE", Description = StoreArgumentDescription)]
+            string home,
+            [Argument("OFFSET", Description = "block index")]
+            int offset,
+            [Argument("LIMIT", Description = "block index")]
+            int limit
+        )
+        {
+            var store = LoadStoreFromUri(home);
+            var prev = DateTime.UtcNow;
```

Comment:
`DateTimeOffset` is better than `DateTime` because it has a timezone offset too. (like aware datetime in python)
```suggestion
            DateTimeOffset prev = DateTimeOffset.UtcNow;
```

Or, we can use [`Stopwatch`](https://docs.microsoft.com/en-US/dotnet/api/system.diagnostics.stopwatch?view=net-5.0) for this purpose.